moveit_ros/visualization/motion_planning_rviz_plugin/src/motion_planning_display.cpp

```diff
@@ -1430,6 +1430,9 @@ void MotionPlanningDisplay::fixedFrameChanged()
   PlanningSceneDisplay::fixedFrameChanged();
   if (int_marker_display_)
     int_marker_display_->setFixedFrame(fixed_frame_);
+  // When the fixed frame changes we need to tell RViz to update the rendered interactive marker display
```

Comment:
Shouldn't this be part of `InteractiveMarkerDisplay::fixedFrameChanged()`? Btw, should we switch back to the [upstream implementation](https://github.com/ros2/rviz/blob/ros2/rviz_default_plugins/include/rviz_default_plugins/displays/interactive_markers/interactive_marker_display.hpp)?

Reply:
`scene_marker_`'s type is InteractiveMarker not InteractiveMarkerDisplay :D, it's already the case for `InteractiveMarkerDisplay::fixedFrameChanged()`

> Btw, should we switch back to the upstream implementation?

Yes, that's right, I'll work on pushing it upstream but that I think we need a multi thread executor for it to work https://github.com/ros2/rviz/blob/ros2/rviz_common/include/rviz_common/visualization_manager.hpp#L400

Reply:
Stumbling over this again... since `requestPoseUpdate()` just feeds in the existing poses, I'd say that [updateReferencePose()](https://github.com/ros2/rviz/blob/cd1da34cfc01d9c6223c181ba6e7bf277ee4c1a8/rviz_default_plugins/src/rviz_default_plugins/displays/interactive_markers/interactive_marker.cpp#L600) is more appropriate here.